deploy/docs/v2_migration_doc.md

```diff
@@ -136,7 +157,39 @@ kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheu
 kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.43/example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml
 ```
 
-### 3. Run upgrade script
+### 3. Prepare temporary instance of fluent-bit
+
+As `spec.selector` in fluent-bit Helm chart was modified it is required to delete existing DaemonSet with old version of `spec.selector` before upgrade. To avoid of data loss a new temporary instance of fluent-bit needs to be created.
+
+To create a temporary copy of fluent-bit DaemonSet:
+
+```bash
+kubectl get daemonset -n <NAMESPACE-NAME> -l "app=fluent-bit,release=<RELEASE-NAME>" -o yaml | \
+yq w - "items[*].metadata.name" "tmp-fluent-bit" | \
+yq w - "items[*].metadata.labels[app.kubernetes.io/name]" "fluent-bit" | \
+yq w - "items[*].metadata.labels[app.kubernetes.io/instance]" "<RELEASE-NAME>" | \
+yq w - "items[*].metadata.labels[heritage]" "tmp" | \
+yq w - "items[*].spec.template.metadata.labels[app.kubernetes.io/name]" "fluent-bit" | \
+yq w - "items[*].spec.template.metadata.labels[app.kubernetes.io/instance]" "<RELEASE-NAME>" | \
+yq w - "items[*].spec.template.metadata.labels[heritage]" "tmp" | \
+kubectl create -f -
```

Comment:
We should add requirements section, so everyone knows which `yq`, etc is needed

Reply:
This has been added to v2 upgrade script https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/170d7249ac0bf71e01c7b103082442fa9d5d8fce/deploy/helm/sumologic/upgrade-2.0.0.sh#L126-L131

and it's also mentioned in `How to upgrade -> Requirements` section of this PR